File: camp/week1/how-many-numbers-are-smaller-than-the-current-number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int>x;
        for(int i=0;i<nums.size();i++){
           int counter=0;
            for(int j=0;j<nums.size();j++){
               if(nums[i]>nums[j]) counter++;
            }
            x.push_back(counter);
        }
        return x;
    }
};

File: camp/week1/leetcode/how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        l=nums[:]
        ans=[]
        for n in range(len(nums)):
            mini=n
            for j in range(n+1,len(nums)):
                if nums[j]<nums[mini]:
                    mini=j
            nums[n],nums[mini]=nums[mini],nums[n]
        for i in range(len(nums)):
            ans+=[nums.index(l[i])]
        return ans
            

        